clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -6113,9 +6113,13 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
   if (KernelOrKext && RawTriple.isOSDarwin())
     CmdArgs.push_back("-fforbid-guard-variables");
 
-  if (Args.hasFlag(options::OPT_mms_bitfields, options::OPT_mno_ms_bitfields,
-                   Triple.isWindowsGNUEnvironment())) {
-    CmdArgs.push_back("-mms-bitfields");
+  if (Args.hasArg(options::OPT_mms_bitfields) ||
```

Comment:
If the LangOption default varies across targets, and you want to forward the driver option to clang -cc1,

```
if (auto *A = Args.getLastArg(..., ...)) {
  if (A->matches(...)) CmdArgs.push_back...
  else CmdArgs.push_back...
}
```

Reply:
@DanShaders Sorry for the bother, if really do not have time to continue it, I'd like to continue it with new MR>